index.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link rel="icon" class="object-contain" href="../image/gsa_super_g-64.gif" type="image/icon type">
+<title>Google</title>
+
+<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap" rel="stylesheet">
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+    <link href="./dist/output.css" rel="stylesheet" />
+  </head>
+  <body class="relative flex flex-col gap-8 h-screen font-[Roboto] text-[0.3rem] md:text-base text-gray-600 w-full ">
+    <nav class="w-full flex gap-2 lg:gap-8 p-1 lg:p-4  justify-end items-center">
+      <a class="hover:underline" href="/">Gmail</a>
+      <a class="hover:underline" href="/">Images</a>
+      <div class="hover:bg-gray-100 inline-flex rounded-full h-3 W-3 lg:h-10 object-center items-center justify-center lg:w-10">
+        <span class="material-symbols-outlined object-contain">
+          apps
+          </span>
+      </div>
+      <div class="hover:bg-gray-100 p-1 rounded-full">
+      <img
+        class="h-4 lg:h-8 rounded-full"
+        src="https://lh3.googleusercontent.com/ogw/AOh-ky1NHTAUFnai-zTmcO6Rsy0YYjrWXaghgfmmp4oB=s64-c-mo"
+        alt=""
+      /></div>
+     </div>
+    </nav>
+    <div class="w-full max-w-full mx-auto flex-1 flex flex-col gap-6 pt-16 sm:pt-32 md:pt-24 lg:pt-0 xl:pt-0">
+      <img
+        class="object-contain h-8 lg:h-[5.5rem]"
+        src="/image/google-lg.png"
+        alt=""
+      />
+     
+      <div
+        class="flex items-center w-[586px]  max-w-[70%] mx-auto justify-between bg-white border-[1px] rounded-full border-gray-200 hover:shadow-xl h-[30px] lg:h-[45px] px-2"
+      >
+        <span class="material-symbols-outlined h3 lg:h-6 m-auto text-gray-400">
+          search
+        </span>
+        
+        <input class="mx-2 h-4  lg:h-8 w-full outline-none" type="text" name="" id="" />
+        <img
+          class="h-4 lg:h-8 object-cover"
+          src="/image/voice-removebg-preview.png"
+          alt=""
+        />
+      </div>
```

Comment:
This input section should be wrapped in a form tag to follow the html good practice